runtime/vara/src/lib.rs

```diff
@@ -270,8 +270,7 @@ impl pallet_babe::Config for Runtime {
     type WeightInfo = ();
     type MaxAuthorities = MaxAuthorities;
 
-    type KeyOwnerProof =
-        <Historical as KeyOwnerProofSystem<(KeyTypeId, pallet_babe::AuthorityId)>>::Proof;
+    type KeyOwnerProof = sp_session::MembershipProof;
```

Comment:
what if it gonna change in future, it's easy to miss it now

Reply:
just copy this from polkadot rococo-runtime if this has changed
https://github.com/paritytech/polkadot-sdk/pull/2644